README.md

```diff
@@ -36,3 +36,9 @@ So, make sure you have Hilt & kapt imported in your projects via below check-lis
    }
    ```
 7. Now you're ready to use the Text to Speech function
+
+## How to release a new version?
+
+1. Update versionCode, versionName in `utils/build.gradle` file and merge to `main`
+2. Trigger `Release` task in Github Actions on `main` branch
```

Comment:
```suggestion
2. Trigger [`Release`](https://github.com/elimu-ai/common-utils/actions/workflows/gradle-release.yml) task in Github Actions on `main` branch
```

Reply:
@tuancoltech I suggest adding this link to the workflow, so that a new person reading this won't have to spend time searching for it.

Reply:
@jo-elimu Sure, I've pushed it.